content/en/guide/v10/forms.md

```diff
@@ -83,7 +83,7 @@ class MySelect extends Component {
   render(_, { value }) {
     return (
       <form onSubmit={this.onSubmit}>
-        <select value={value} onInput={this.onInput}>
+        <select value={value} onChange={this.onInput}>
```

Comment:
Should I change the name of the method as well?

Reply:
yeah, lets.

Reply:
Done